CHANGELOG.md

```diff
@@ -1,16 +1,27 @@
 # Head
 
-
--   Deprecated: `block-no-single-line`. Use `block-opening-brace-newline-after` and `block-closing-brace-newline-before` rules with the option `"always"` instead. See [the FAQs for an example](docs/user-guide/faq.md#how-do-i-disallow-single-line-blocks).
--   Deprecated: `declaration-block-no-ignored-properties`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `declaration-block-properties-order`. Use the community [`stylelint-order`](https://github.com/hudochenkov/stylelint-order) plugin pack instead. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `media-feature-no-missing-punctuation`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `selector-no-empty`. If you use this rule, please consider creating a plugin for the community. See the [release planning](docs/user-guide/release-planning.md) document for more details.
--   Deprecated: `"all-nested"` option for `at-rule-empty-line-before`. Use the `"inside-block"` option instead.
--   Deprecated: `"blockless-group"` option for `at-rule-empty-line-before`. Use the `"blockless-after-blockless"` option instead.
--   Deprecated: `"between-comments"` option for `comment-empty-line-before`. Use the `"after-comment"` option instead.
--   Deprecated: `"stylelint-commands"` option for `comment-empty-line-before`. Use the `"stylelint-command"` option instead.
--   Deprecated: `"at-rules-without-declaration-blocks"` option for `max-nesting-depth`. Use the `"blockless-at-rules"` option instead.
+See the [release planning](docs/user-guide/release-planning.md) document for information on some of these rule deprecations.
+
+-   Deprecated: 13 rules ([#2197](https://github.com/stylelint/stylelint/pull/2197) & [#2285](https://github.com/stylelint/stylelint/pull/2285)).
```

Comment:
> because these are major changes and users will want to know what's going on

Agreed, but I thought that is what the [release planning document](https://github.com/stylelint/stylelint/blob/cl-and-rp/docs/user-guide/release-planning.md) was for? i.e. a canonical place we can keep up to date with evolving suggestions about what to do leading up a release. 

> For example, all of the spec validation rules could be grouped with a statement like this:

Agreed. At the moment I've split out the wrapping rules into their own group and kept `declaration-block-properties-order` by itself. Splitting out the validation rules into their own group makes sense. Any other suggestions?